File: Pizza Wahala/pizza_wahala.py
sapa_slice = 4
sapa_price = 2000
small_money_slice = 6
small_money_price = 2400
big_boys_slice = 8
big_boys_price = 3000
odogwu_slice = 12
odogwu_price = 4200

print("""--------Menu-------
Sapa size
Sapa_slice = 4
Sapa_price = 2000

Small Money size
Small_money_slice = 6
Small_money_price = 2400

Big Boys size
Big_boys_slice = 8
Big_boys_price = 3000

Odogwu size
Odogwu_slice = 12
Odogwu_price = 4200
 """)
number_of_people = int(input("How Many Guests Do You Intend Serving: "))
pizza_type = input("What Type Of Pizza Do You Intend Buying: ").lower()

if pizza_type not in ["small money size","sapa size","big boys size","odogwu size" ]  or number_of_people <= 0:
    print("invalid Selection")
    exit()

if pizza_type == "sapa size" and number_of_people % sapa_slice == 0:
    pizza_box =  number_of_people / sapa_slice
    amount_to_pay = sapa_price * pizza_box
    leftover = 0
elif pizza_type == "sapa size" and number_of_people % sapa_slice != 0:
    pizza_box =  (number_of_people // sapa_slice) + 1
    amount_to_pay = sapa_price * pizza_box
    leftover = (pizza_box * sapa_slice) - number_of_people

if pizza_type == "small money size" and number_of_people % small_money_slice == 0:
    pizza_box =  number_of_people / small_money_slice
    amount_to_pay = small_money_price * pizza_box
    leftover = 0
elif pizza_type == "ssmall money size" and number_of_people % small_money_slice != 0:
    pizza_box =  (number_of_people // small_money_slice) + 1
    amount_to_pay = small_money_price * pizza_box
    leftover = (pizza_box * small_money_slice) - number_of_people

if pizza_type == "big boys size" and number_of_people % big_boys_slice == 0:
    pizza_box =  number_of_people / big_boys_slice
    amount_to_pay = big_boys_price * pizza_box
    leftover = 0
elif pizza_type == "big boys size" and number_of_people % big_boys_slice != 0:
    pizza_box =  (number_of_people // big_boys_slice) + 1
    amount_to_pay = big_boys_price * pizza_box
    leftover = (pizza_box * big_boys_slice) - number_of_people

if pizza_type == "odogwu size" and number_of_people % odogwu_slice == 0:
    pizza_box =  number_of_people / odogwu_slice
    amount_to_pay = odogwu_price * pizza_box
    leftover = 0
elif pizza_type == "odogwu size" and number_of_people % odogwu_slice != 0:
    pizza_box =  (number_of_people // odogwu_slice) + 1
    amount_to_pay = odogwu_price * pizza_box
    leftover = (pizza_box * odogwu_slice) - number_of_people

print("Number of boxes of pizza to buy = ",int(pizza_box), "boxes") 
print("Number of left over slice after serving = ",leftover, "slices")
print("Amount= ", amount_to_pay)

File: Pizza Wahala/pizzawahala.js
const prompt = require("prompt-sync")();

const sapaSlice = 4;
const sapaPrice = 2000;

const smallMoneySlice = 6;
const smallMoneyPrice = 2400;

const bigBoysSlice = 8;
const bigBoysPrice = 3000;

const odogwuSlice = 12;
const odogwuPrice = 4200;

console.log(`
--------Menu-------
Sapa size
Sapa_slice = 4
Sapa_price = 2000

Small Money size
Small_money_slice = 6
Small_money_price = 2400

Big Boys size
Big_boys_slice = 8
Big_boys_price = 3000

Odogwu size
Odogwu_slice = 12
Odogwu_price = 4200
`);


let numberOfPeople = parseInt(prompt("How Many Guests Do You Intend Serving: "));
let pizzaType = prompt("What Type Of Pizza Do You Intend Buying: ").toLowerCase();

if (
  !["sapa size", "small money size", "big boys size", "odogwu size"].includes(pizzaType) ||
  numberOfPeople <= 0
) {
  console.log("Invalid Selection");
  process.exit(0);
}

let pizzaBox = 0;
let amountToPay = 0;
let leftover = 0;

if (pizzaType === "sapa size" && numberOfPeople % sapaSlice === 0) {
  pizzaBox = numberOfPeople / sapaSlice;
  amountToPay = sapaPrice * pizzaBox;
  leftover = 0;
}
else if (pizzaType === "sapa size" && numberOfPeople % sapaSlice !== 0) {
  pizzaBox = Math.floor(numberOfPeople / sapaSlice) + 1;
  amountToPay = sapaPrice * pizzaBox;
  leftover = (pizzaBox * sapaSlice) - numberOfPeople;
}


else if (pizzaType === "small money size" && numberOfPeople % smallMoneySlice === 0) {
  pizzaBox = numberOfPeople / smallMoneySlice;
  amountToPay = smallMoneyPrice * pizzaBox;
  leftover = 0;
}
else if (pizzaType === "small money size" && numberOfPeople % smallMoneySlice !== 0) {
  pizzaBox = Math.floor(numberOfPeople / smallMoneySlice) + 1;
  amountToPay = smallMoneyPrice * pizzaBox;
  leftover = (pizzaBox * smallMoneySlice) - numberOfPeople;
}


else if (pizzaType === "big boys size" && numberOfPeople % bigBoysSlice === 0) {
  pizzaBox = numberOfPeople / bigBoysSlice;
  amountToPay = bigBoysPrice * pizzaBox;
  leftover = 0;
}
else if (pizzaType === "big boys size" && numberOfPeople % bigBoysSlice !== 0) {
  pizzaBox = Math.floor(numberOfPeople / bigBoysSlice) + 1;
  amountToPay = bigBoysPrice * pizzaBox;
  leftover = (pizzaBox * bigBoysSlice) - numberOfPeople;
}


else if (pizzaType === "odogwu size" && numberOfPeople % odogwuSlice === 0) {
  pizzaBox = numberOfPeople / odogwuSlice;
  amountToPay = odogwuPrice * pizzaBox;
  leftover = 0;
}
else if (pizzaType === "odogwu size" && numberOfPeople % odogwuSlice !== 0) {
  pizzaBox = Math.floor(numberOfPeople / odogwuSlice) + 1;
  amountToPay = odogwuPrice * pizzaBox;
  leftover = (pizzaBox * odogwuSlice) - numberOfPeople;
}

console.log("Number of boxes of pizza to buy =", pizzaBox, "boxes");
console.log("Number of left over slice after serving =", leftover, "slices");
console.log("Amount =", amountToPay);





